### PR TITLE
Refactor of Simple Animal Eating

### PR DESCRIPTION
**Pull request by Fox-McCloud**
What started off as someone poking me to allow Vulpkanin to eat chickens....turned into a refactor of eating of simple animals in relation to particular species.

Simple animal eating (or any mob, really) is now governed by a single var on each species. In addition to this, you can specify a list of "legal" mobs that that species can eat---yes, you can technically specify other non-simple animal mobs, but that's generally not a good idea. On the whole this means less snowflaking of mob checks in grab code and allows for much easier expansion in the future.

Changes
- Tajarans can now eat mice, chicks, parrots, and tribbles..generally smallish prey type animals
- Vulpkanin and Unathi can eat mice, chicks, lizards, chickens, crabs, parrots, and tribbles...due to their superior strength and strongest jaws, both these races can eat larger prey.
- Kidan can still only eat nymphs

Fixes
- Fixes simple animals never dying in your stomach, digesting, or giving you nutrition

:cl: Fox McCloud
add: Tajarans can now eat mice, chicks, parrots, and tribbles
add: Vulpkanin and Unathi can now eat mice, chicks, lizards, chickens, crabs, parrtos, and tribbles
/:cl:

### CODE REVIEW

**Comment by Keilden** [comment]
IPCs eating drone to upgrade themself, slimes eating slimes to get buffed.

**Comment by FlattestGuitar** [comment]
Rip Poly

**Comment by ProperPants** [comment]
>IPCs eating drone to upgrade themself

IPCs being allowed to slot to E-N and steal his power until he dies.

**Comment by KasparoVy** [comment]
Wiiicked.

Lmao, E-N would be kill every single round then

**Comment by QQCucumber** [comment]
Oh yeah! We're one step closer to becoming vorestation.

**Comment by CrAzYPiLoT-SS13** [comment]
I kinda agree with the last comment, but I also like it. :+1::-1:

**Comment by ProperPants** [comment]
>Lmao, E-N would be kill every single round then

That's the point :wink:

**Comment by Tastyfish** [comment]
Never eating dead flesh again

File: code/modules/mob/living/carbon/human/species/species.dm [hunk 1]
```diff
@@ -63,6 +63,8 @@
 
 	var/ventcrawler = 0 //Determines if the mob can go through the vents.
 	var/has_fine_manipulation = 1 // Can use small items.
+	var/can_consume_mobs = 0 //can it consume mobs?
```
Comment (Tastyfish) [inline code/modules/mob/living/carbon/human/species/species.dm]: Isn't this var redundant? This list could just be blank.

**Comment by Kevak** [comment]
I approve of this whole heartedly. Now nooooobody can find the evidence when I off Poly for being annoying.

**Pull request merged**